.Lib9c.StateService/Program.cs

```diff
@@ -1,10 +1,22 @@
 using Bencodex;
+using Destructurama;
 using Libplanet.Action.State;
 using Libplanet.Extensions.RemoteBlockChainStates;
+using Serilog;
+using Serilog.Formatting.Compact;
 
 var builder = WebApplication.CreateBuilder(args);
 
 builder.Configuration.AddEnvironmentVariables();
+Log.Logger = new LoggerConfiguration()
+    .WriteTo.File(
+        new RenderedCompactJsonFormatter(),
+        path: Environment.GetEnvironmentVariable("JSON_LOG_PATH") ?? "./logs/remote-headless_9c-network_remote-headless.json",
```

Comment:
I'm not sure about the default log path but it can be merged now if you're busy.

Reply:
https://github.com/planetarium/lib9c/pull/2063/commits/97bd3b06113ccf20d49a75d653da095e633ca114 changed.